README.md

```diff
@@ -1,2 +1,39 @@
 # k8s
 Kubernetes/Helm configs for ReportPortal
+
+**Do note that this is a Beta version**
+
+
+This Helm project is created to setup ReportPortal with only one commando.  
+The help chart is tested on Minikube and creates a fully working ReportPortal project
+
+The chart installs all mandatory services to run ReportPortal
+
+The Helm chart installation consist of the following .yaml files:
+
+- Statefulset and Service files of: `Analyzer, Api, Elasticsearch, Gateway, Index, Mongodb, Registry, UAT, UI` that are used for deployment and communication between services.
+- PersistentVolume files: `Elasticsearch, Mongodb, Registry`
+- A `Ingress` to access the UI
+- A `values.yaml` which exposes a few of the configuration options in the
+charts.
+- A `templates/_helpers.tpl` file which contains helper templates. 
+
+
+You can deploy this chart with `helm install ./<project folder>`. 
+
+Once it's installed please make sure that the PersistentVolumes directories are created
+
+Commando to create those folders:
```

Comment:
I suggest that this is not the best option to use hostpath.
I preffer to see helm chart with options:
- Use specific PVC for specific component
- Create a new one with some StorageClass or `default` by default